.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Report an issue with the Wayfair plugin
+title: "[New Bug]"
+labels: bug, new
+assignees: ''
+
+---
+
+**Date first encountered**:
+**Supplier's client ID (if known):**
+**Plentymarkets Version / release channel:**
+
+**Short description:**
+[ie: Cannot save Warehouse mapping]
+
+**To Reproduce**
+Steps to reproduce the behavior (if repeatable, please perform with the Wayfair logging in Plentymarkets set to the `Debug` level):
+1. [ie: Go to Warehouses settings page]
+2. [ie: Add new Warehouse mapping]
+3. [Ie: Click save]
+
+**Expected behavior:**
+[ie: New Warehouse mapping stays on page]
+
+**Actual behavior:**
+[ie: New Warehouse mapping disappears after refreshing the page]
+
+**Screenshots:**
+[If applicable, add screenshots to help explain your problem.]
+
+**Related logs:**
+[Please attach logs as files, and describe them here.]
```

Comment:
Some questions bug-reporting forms ask:

- Plugin version (will they know this from using the Plugin? A fancy version of a bug-submission form could  automatically populate this)
- Operating System
- Browser
- Time of day (can this automatically be generated when user submits form?)

Reply:
- Thanks! Plugin version was in my original draft but it's missing here!
- 
- github will track the creation date of this, but it's not always going to be true that `time of report` is equal to `time it is experienced`
- 
- OS and browser were on the suggested template but I thought it was getting too noisy. will consider bringing them back.